specification/columns/chargedescription.md

```diff
@@ -1,6 +1,6 @@
 # Charge Description
 
-A Charge Description provides a high-level context of a row without requiring additional discovery. This column is a self-contained, summary explanation of what the charge is for and how it was priced. It typically covers a select group of corresponding details across a billing dataset or provides information not otherwise available.
+A Charge Description provides a high-level context of a row without requiring additional discovery. This column is a self-contained summary of what the charge is for and how it is priced. It typically covers a select group of corresponding details across a billing dataset or provides information not otherwise available.
```

Comment:
```suggestion
A Charge Description provides a high-level context of a row without requiring additional discovery. This column is a self-contained summary of the charge's purpose and price. It typically covers a select group of corresponding details across a billing dataset or provides information not otherwise available.
```